nx.json

```diff
@@ -39,12 +39,6 @@
     },
     "lint:check": {},
     "lint:fix": {},
```

Comment:
So looks like nx is not involved with the release process anymore?

Reply:
It's still involved, it's just that these scripts are on longer used.